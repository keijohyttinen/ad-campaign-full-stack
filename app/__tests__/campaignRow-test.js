import 'react-native';
import React from 'react';
import CampaignMainRow from '../src/components/campaignMainRow';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders CampaignMainRow correctly', () => {
  const props = {
    name: "Campaing Row prints TestName-2",

  };
  const navigation = {
    navigate: () => { }
  }
  const tree = renderer.create(<CampaignMainRow {...props} navigation={navigation} />).toJSON();
  expect(tree).toMatchSnapshot();
});
