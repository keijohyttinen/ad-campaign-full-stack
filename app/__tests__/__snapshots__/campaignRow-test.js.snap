// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CampaignMainRow correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "marginBottom": 25,
      "marginLeft": 15,
      "opacity": 1,
    }
  }
>
  <View>
    <Text
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          Object {
            "fontSize": 27.5,
          },
          undefined,
          undefined,
          undefined,
          Object {},
          undefined,
          undefined,
        ]
      }
    >
      Campaing Row prints TestName-2
    </Text>
    <Text
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      Goal: undefined
    </Text>
    <Text
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      Budget: undefined
    </Text>
    <Text
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      Status: undefined
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  />
  <View
    style={
      Object {
        "marginRight": 15,
      }
    }
  >
    <Text
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      &gt;
    </Text>
  </View>
</View>
`;
